Content.IntegrationTests/GameTests.cs

```diff
@@ -0,0 +1,27 @@
+using System.Diagnostics;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using OpenDreamRuntime;
+using Robust.Server.Player;
+using Robust.Shared.IoC;
+using Robust.Shared.Network;
+
+namespace Content.IntegrationTests {
+    [TestFixture]
+    public sealed class GameTests : ContentIntegrationTest {
+
+        /// <summary>
+        /// Tests to make sure the IntegrationTest project does not runtime.
+        /// </summary>
+        [Test]
+        public async Task NoRuntimesTest() {
+            var (client, server) = await StartConnectedServerClientPair();
+            await RunTicksSync(client, server, 1000);
+            Assert.IsTrue(server.IsAlive);
+            var manager = server.ResolveDependency<IDreamManager>();
+            if(manager.LastDMException is not null) {
```

Comment:
You could instead subscribe to the `IDreamManager.OnException` event so that if multiple exceptions occur they can all be presented.

Reply:
Currently the project is set up to emit pretty exhaustive logs from the server into the console, so all runtimes that occur seems to be displayed just fine.

That's just there to make sure that it causes a test failure.